project/MiMa.scala

```diff
@@ -662,7 +662,18 @@ object MiMa extends AutoPlugin {
         ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.mapError"),
 
         // #21894 Programmatic configuration of the ActorSystem
-        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.ActorSystemImpl.this")
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.ActorSystemImpl.this"),
+
+        // #21213 Feature request: Let BackoffSupervisor reply to messages when its child is stopped
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.pattern.BackoffSupervisor.this"),
```

Comment:
As in this [comment](https://github.com/akka/akka/blob/98c2adfc98f23f461427c674eb78eef1a74b2957/akka-actor/src/main/scala/akka/pattern/BackoffSupervisor.scala#L168L169), since this won't be called directly it is ok?

```
/**
 * ...
 * This back-off supervisor is created by using `akka.pattern.BackoffSupervisor.props`
 * with `Backoff.onStop`.
 */
```

Anyway I preserved the signature of this constructor, so not sure why I got this bin-compat error..
```
  def this(
    childProps:         Props,
    childName:          String,
    minBackoff:         FiniteDuration,
    maxBackoff:         FiniteDuration,
    randomFactor:       Double,
    supervisorStrategy: SupervisorStrategy) =
    this(childProps, childName, minBackoff, maxBackoff, AutoReset(minBackoff), randomFactor, supervisorStrategy, None)
```